mitodl/mitxonline
# feat: Add Certificate Template using Wagtail CMS

#### Pre-Flight checklist


#### What are the relevant tickets?
fixes: #691 

#### What's this PR do?
The PR enables the wagtail part to setup certificates along with signatories.

#### How should this be manually tested?
- Checkout the branch.
- Run the migrations
- Create a super user and visit `CMS`.
- In Pages > HomePages > ADD CHILD PAGE to ensure certificates and signatories pages in wagtail.
- In Courses select course and add certificate, at bottom click on preview to get the preview of the certificate

#### Screenshots

![screencapture-localhost-8013-cms-pages-13-edit-2022-07-25-10_59_31](https://user-images.githubusercontent.com/7334669/180941215-9323bf8f-38a9-41f6-a98d-ad010247b089.png)
![screencapture-localhost-8013-cms-pages-add-cms-signatorypage-10-2022-07-25-10_58_59](https://user-images.githubusercontent.com/7334669/180941232-9afeb046-e142-4e5e-89e8-1dfc3c4cc04a.png)
![screencapture-localhost-8013-cms-pages-3-2022-07-25-10_58_08](https://user-images.githubusercontent.com/7334669/180941236-11f19655-e286-423a-861d-c87a1d0e2009.png)

#### Demo Certificate Template Screenshot
![screencapture-localhost-8013-cms-pages-44-demo-certificate-2022-07-29-04_42_43_PM](https://user-images.githubusercontent.com/77275478/181754106-c4e18e48-956d-49c5-a557-51317e72858f.png)
